src/components/KonnectorSuccess.jsx

```diff
@@ -0,0 +1,34 @@
+import styles from '../styles/accountConnection'
+
+import React from 'react'
+import { translate } from '../plugins/i18n'
+
+import DescriptionContent from './DescriptionContent'
+import AccountLoginForm from './AccountLoginForm'
+
+export const KonnectorSuccess = ({ t, success, connector, isTimeout, folderPath, onAccountConfig, onCancel }) => {
+  return (
+    <div>
+      <DescriptionContent
+        title={t(`account.success.title.${success.type}`, { name: connector.name })}
+        messages={success.messages}
+      >
+        { Array.isArray(connector.dataType) && connector.dataType.includes('bill') &&
+          <p>
+            {t(`account.message.${isTimeout ? 'syncing' : 'synced'}.bill`, { name: connector.name })}
+            <br />
+            <span className={styles['col-account-success-highlighted-data']}>{folderPath}</span>
+          </p>
+        }
+      </DescriptionContent>
+      // FIXME: Why the hell do we use an AccountLoginForm for redirect?
```

Comment:
Now you can add all [these stuffs](https://github.com/cozy/cozy-collect/blob/master/src/components/AccountLoginForm.jsx#L99) juste here instead of the AccountLoginForm, can't you?